provisioning/buffer/kubernetes/templates/api/deployment.yaml

```diff
@@ -0,0 +1,120 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: buffer-api
+  namespace: $NAMESPACE
+  labels:
+    app: buffer-api
+    releaseId: $RELEASE_ID
+spec:
+  replicas: $BUFFER_API_REPLICAS
+  selector:
+    matchLabels:
+      app: buffer-api
+  template:
+    metadata:
+      labels:
+        app: buffer-api
+        releaseId: $RELEASE_ID
+        buffer-etcd-client: "true"
+        tags.datadoghq.com/env: "$KEBOOLA_STACK"
+        tags.datadoghq.com/service: "buffer-api"
+        tags.datadoghq.com/version: "$RELEASE_ID"
+      annotations:
+        log: "true"
+        ad.datadoghq.com/tags: '{"git.commit.sha": "$COMMIT_HASH", "git.repository_url": "https://github.com/keboola/keboola-as-code"}'
+    spec:
+      terminationGracePeriodSeconds: 600
```

Comment:
Pri nejakom uplnom vypadku `etcd` si API node drzi prijate zaznamy v memory a snazi sa ich ulozit do `etcd`, preto `GracePeriod = 10m`, to iste worker, ten zase nahrava subory do `file storage`, co moze trvat par minut, kym as upload dokonci.